src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -108,31 +114,41 @@ export function PreviewSiteRow( {
 		<div className="self-stretch flex-col">
 			<div className="flex items-center px-8 py-6">
 				<div className="w-[51%] overflow-hidden pe-4">
-					<div className="flex items-center">
-						<div
-							className={ cx(
-								'text-[13px] leading-5 line-clamp-1 break-all',
-								isExpired && 'line-through text-a8c-gray-700'
-							) }
-						>
-							{ /* translators: %s: Site name (e.g. "My Site Preview") */ }
-							{ snapshot.name || sprintf( __( '%s Preview' ), selectedSite.name ) }
-						</div>
-					</div>
-					<Button
-						variant="link"
-						disabled={ isExpired }
-						className={ cx(
-							'!text-a8c-gray-700 max-w-full',
-							isExpired ? 'pointer-events-none' : 'hover:!text-a8c-blue-50'
+					<Tooltip
+						text={ __(
+							'This preview site has been marked as deleted. You can no longer access it. You can clear it from you list by clicking the Clear button.'
```

Comment:
```suggestion
							'This preview site has been deleted on the server. Remove it from the list by clicking the Clear button.'
```
Just a suggestion. Making it briefer seems like an improvement.

Reply:
Looks good to me. I have updated the message.